src/CsvTextEditor/ProjectManagement/Watchers/CsvTextEditorIsDirtyProjectWatcher.cs

```diff
@@ -59,6 +76,23 @@ protected override Task OnActivatedAsync(IProject oldProject, IProject newProjec
             return base.OnActivatedAsync(oldProject, newProject);
         }
 
+        private void AutoSaveIfNeeded(object sender, EventArgs e)
+        {
+            if (!_configurationService.IsRoamingValueAvailable(Configuration.TimedAutoSave))
+                return;
```

Comment:
I don't think this needs to be here. Just use:

var autoSave = GetRoamingValue(configuration.AutoSaveEditor, configuration.AutoSaveEditorDefaultValue)